### ISSUE TITLE
リグレッションテスト vs. 回帰テスト

### ISSUE THREAD

**Issue by tatsuo-ishii** [pgsql-jp/jpug-doc#2700]
### 問題の詳細

regression testの訳語が「リグレッションテスト」と「.回帰テスト」で混在しています。数としては「リグレッションテスト」(64)、「回帰テスト」(3)で、リグレッションテストの方が圧倒的に多いようです。
「リグレッションテスト」に統一で良いでしょうか？

### 貢献者として記載可否

記載（貢献者欄に書いてください）

**Comment by KenichiroTanaka** [comment]
はい、普段使っている用語としては、リグレッションテストが多く、私も「回帰テスト」より「リグレッションテスト」の方が良いように思います。

**Comment by koizumistr** [comment]
「日本語ドキュメントの中での多数派に揃える」に賛成です。


「回帰テスト」も「リグレッションテスト」も普段、仕事では（言葉としては）使わないので、世間一般ではどうなっているのか調べてみました。
- "回帰テスト" 約 23,400 件 
- "リグレッションテスト" 約 28,000 件

となりましたので、世間一般でもどちらが一般的という訳ではなさそうです。ですので、これは日本語ドキュメントの中での決めの問題ということになり、（圧倒的）多数派に揃えるというのが妥当だと思います。

**Comment by tatsuo-ishii** [comment]
「リグレッションテスト」に統一で、pull requestを出しておきます。

**Comment by koizumistr** [comment]
覚書
#2708 にて対応中
